docs/admin-manual/tutorials/setup-liquidsoap-aac-streaming.md

```diff
@@ -26,18 +26,32 @@ For Ubuntu 20.04 LTS ('focal'), use the following file:
 wget https://github.com/savonet/liquidsoap/releases/download/v1.4.4/liquidsoap-v1.4.4_1.4.4-ubuntu-focal-amd64-1_amd64.deb
 ```
 
-For Debian 11 ('Bullseye'), use the following file:
+For Debian 11 ('Bullseye'), use the following files: (Debian 11 ships fdk-aac2, so you'll need to download fdk-aac1 from Debian 10)
```

Comment:
Could link me where you found out that fdk-aac1 is required instead of fdk-aac2 ? I've used the one in the non-free repository without any problem.

Why did you not propose to enable the non free repository ?

Reply:
The error message when installing the liquidsoap 1.4.4 deb prompted me. I just tried installing libfdk-aac2 instead of 1, but as soon as i try to uninstall 1 it wants to remove the new liquidsoap as well.